ground/gcs/share/taulabs/pfd/default/PfdIndicators.qml

```diff
@@ -39,14 +39,13 @@ Item {
     // GPS status text
     Text {
         id: gps_text
-        text: "GPS: " + GPSPosition.Satellites + "\nPDP: " + GPSPosition.PDOP.toFixed(2) + "\nACC: " + GPSPosition.Accuracy.toFixed(2)
+        text: "GPS: " + GPSPosition.Satellites.charCodeAt(0) + "\nPDP: " + GPSPosition.PDOP.toFixed(2) + "\nACC: " + GPSPosition.Accuracy.toFixed(2)
```

Comment:
This takes the first character of the "Satellites" string and takes its ascii value.  Unfortunately because of the upstream QT bug of treating any character types as a string we have to do this ordinal value here.

It works for values 0..127, maybe for 0..255.